sdk/resource/resource_test.go

```diff
@@ -260,3 +265,174 @@ func TestMarshalJSON(t *testing.T) {
 		`[{"Key":"A","Value":{"Type":"INT64","Value":1}},{"Key":"C","Value":{"Type":"STRING","Value":"D"}}]`,
 		string(data))
 }
+
+func TestNewRawResource(t *testing.T) {
```

Comment:
nit, but it sounds like we switched from `Raw` to `Empty`.

```suggestion
func TestNewEmptyResource(t *testing.T) {
```